packages/quantic/force-app/main/default/lwc/quanticResultHighlightedTextField/quanticResultHighlightedTextField.css

```diff
@@ -0,0 +1,7 @@
+.result-text__label {
+    font-weight: var(--lwc-fontWeightBold, 700);
+}
+
+b {
+  font-weight: bolder;
```

Comment:
why bolder?

Reply:
So it always takes a value relatively bolder relatively to the parent element: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#bolder